common.blocks/menu-item/menu-item.bemhtml

```diff
@@ -2,7 +2,20 @@ block('menu-item')(
     js()(function() {
         return { val : this.ctx.val };
     }),
-    attrs()({ role : 'menuitem' }),
+    attrs()(function(){
+        var mods = this.mods,
+            menuMode = this._menuMods && this._menuMods.mode,
+            role = menuMode?
+                        (menuMode === 'check'? 'menuitemcheckbox' : 'menuitemradio') :
+                        'menuitem',
+            attrs = {
+                role : role,
+                'aria-disabled' : mods.disabled,
```

Comment:
But if block will be without `disabled` mod `'aria-disabled'` attribute will not be added. So, this attribute will be added just in case when `disabled` mod will be `true`